salt/srv/salt/base/calamari-crush-location.py

```diff
@@ -20,14 +20,14 @@ def get_last_crush_location(osd_id):
     key = 'daemon-private/osd.%s/v1/calamari/osd_crush_location' % osd_id
     osd_keyring = '/var/lib/ceph/osd/ceph-%s/keyring' % osd_id
     admin_keyring = '/etc/ceph/ceph.client.admin.keyring'
-    commands = (['sudo', 'ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
+    commands = (['ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
```

Comment:
missing admin.keyring fails like so:
ERROR:calamari_osd_location:Error 1 running ceph config-key get:'2015-05-26 13:08:18.799376 7faf8b18a700 -1 monclient(hunting): ERROR: missing keyring, cannot use cephx for authentication
2015-05-26 13:08:18.799377 7faf8b18a700  0 librados: client.admin initialization error (2) No such file or directory
Error connecting to cluster: ObjectNotFound'

This causes the script to fall back to host=hostname -s in get_osd_location.

I will audit ceph.py for simmilar issue.